tests/integration/prepare_test_cluster.go

```diff
@@ -0,0 +1,106 @@
+package integration
+
+import (
+	"bytes"
+	"fmt"
+	"io"
+	"os/exec"
+	"path/filepath"
+)
+
+var (
+	waitForDeploymentArgs = []string{
+		"-n",
+		"profiles-system",
+		"wait",
+		"--for=condition=available",
+		"deployment",
+		"profiles-controller-manager",
+		"--timeout",
+		"5m",
+	}
+	waitForPodsArgs = []string{
+		"-n",
+		"profiles-system",
+		"wait",
+		"--for=condition=Ready",
+		"--all",
+		"pods",
+		"--timeout",
+		"5m",
+	}
+	applySourceCatalogArgs = []string{
+		"apply",
+		"-f",
+		filepath.Join("..", "..", "dependencies/profiles/examples/profile-catalog-source.yaml"),
+	}
+)
+
+// PrepareTestCluster will create a test cluster using pctl `prepare` command.
+// @binary -- location of the built pctl binary.
+func PrepareTestCluster(binaryPath string) error {
+	cmd := exec.Command(binaryPath, "prepare", "--dry-run")
+	output, err := cmd.CombinedOutput()
+	if err != nil || !bytes.Contains(output, []byte("kind: List")) {
+		fmt.Println("Output of prepare was: ", string(output))
+		return fmt.Errorf("failed to run prepare command: %w", err)
+	}
+	fmt.Println("Install file generated successfully.")
+
+	fmt.Print("Replacing controller image to localhost:5000...")
+	output = bytes.ReplaceAll(output, []byte("image: weaveworks/profiles-controller:latest"), []byte("image: localhost:5000/profiles-controller:latest"))
```

Comment:
@Skarlso Note to self, I need to replace this with a regex matcher, since this will now no longer be `latest` after https://github.com/weaveworks/profiles/pull/105 is merged.